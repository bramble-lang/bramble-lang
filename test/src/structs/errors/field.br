fn my_main() {
    let t: MyStruct := MyStruct{
        x: 5, 
        y: 1, 
        bad: 12,
    };
    printiln t.x;
    printiln t.y;
    printbln t.z;
    
    return;
}

struct MyStruct {
    x: i32,
    y: i32,
    z: bool,
}