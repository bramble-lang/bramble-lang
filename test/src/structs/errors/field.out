Error: Semantic: L2: member bad not found on root::MyStruct
